src/dmd/backend/dt.c

```diff
@@ -325,7 +325,7 @@ void DtBuilder::dword(int value)
 /***********************
  * Write a size_t value.
  */
-void DtBuilder::size(d_ulong value)
+void DtBuilder::size(unsigned long value)
```

Comment:
This is replacing a 64 bit integer with a 32 bit integer. It cannot work correctly.